pkg/apis/servicecatalog/types.go

```diff
@@ -449,5 +449,13 @@ type Parameters struct {
 	SecretRef *v1.LocalObjectReference
 	// Reference to a secret key in the namespace.
 	// The parameters will be set from the JSON contained in this secret key
-	SecretKeyRef *v1.SecretKeySelector
+	SecretKeyRef *SecretKeyReference
+}
+
+// SecretKeyReference references a key of a Secret.
+type SecretKeyReference struct {
+	// The name of the secret in the pod's namespace to select from.
+	v1.LocalObjectReference
```

Comment:
Don't do this - `SecretKeyReference` will inherit all the methods from `LocalObjectReference` and you absolutely do not want that.